### DISCUSSION TITLE
Where to get replacement batteries?

### DISCUSSION THREAD

**Discussion by r-bt** [bitcraze/discussions#1622] [Q&A]
I'm looking for replacement batteries but am based in the US so shipping costs from Europe are quite expensive. Is there anywhere in the US I can buy replacement batteries?

**Comment by r-bt** [comment]
Any update on this? So far I've only seen https://www.seeedstudio.com/Crazyflie-2-0-spare-battery-including-charger.html but it's currently only available on backorder?

**Reply by tobbeanton** [reply]
As long as the battery is 1-cell LiPo, same polarity and connector, >15C cont. discharge rate, fits mechanically and is not to heavy it should be fine. Recommended is also to use a battery with a PCM (protection circuit). Looking for "Hubsan x4" replacement batteries might be one way.
